### ISSUE TITLE
[wg/sdw] Spatial Data on the Web Working Group Charter

### ISSUE THREAD

**Issue by bert-github** [w3c/strategy#474]
New charter proposal, reviewers please take note.

# Charter Review

[Proposed SDW WG charter](https://w3c.github.io/charter-drafts/2024/sdw-wg.html)

[diff from charter template](https://services.w3.org/htmldiff?doc1=https://w3c.github.io/charter-drafts/charter-template.html&doc2=https://w3c.github.io/charter-drafts/2024/sdw-wg.html)

[Diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2021%2F10%2Fsdw-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2024%2Fsdw-wg.html)

[chair dashboard](https://www.w3.org/PM/Groups/chairboard.html?gid=wg/sdw)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

* New
 - [ ] New WG 
 - [ ] New IG
 - If this is a new WG or IG charter request, link to [Advance Notice](), and any issue discussion: 

* Existing 
 - [X] Existing WG recharter 
 - [ ] Existing IG recharter
 - If this is a charter extension or revision, any issue discussion:

Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, security, and TAG. **Also add a "card" for this issue to the [Strategy Funnel](https://github.com/w3c/strategy/projects/2).**

Communities suggested for outreach: 

Known or potential areas of concern: 

Where would charter proponents like to see issues raised? (this strategy funnel issue, a different github repo, email, ...): [w3c/charter-drafts](https://github.com/w3c/charter-drafts/issues)

Anything else we should think about as we review? 

Note: proposed chairs should be copied @... on this issue: @ldesousa, @rob-metalinkage

**Comment by svgeesus** [comment]
Overall the charter looks good.

I was unsure what this sentence was intended to mean:

> The following existing specifications are potentially in scope for maintenance: 

1. Each specification **might** be in scope or it might not
2. Each specification **is** in scope, and if it needs maintenance then it may be maintained

**Comment by svgeesus** [comment]
> In particular, it lacks a "motivation and background" section

I see that it does now have that section.

**Comment by himorin** [comment]
no comment or request from i18n

**Comment by plehegar** [comment]
[Advance notice](https://lists.w3.org/Archives/Member/w3c-ac-members/2024JulSep/0034.html)

**Comment by plehegar** [comment]
No comment from PING

**Comment by plehegar** [comment]
Some copy-editing improvements:
* the only sentence in the 3 paragraph Motivation and Background section thats doing any work / needed is “The overarching goal of the SDWWG is to enable spatial data to be integrated within the wider Web of data; developing and maintaining standard patterns and specifications addressing these problems.” (not a normative concern, of course)
* Again, there are abbreviations and references to other specs and technologies all over this charter that are not clear to the reader. I have no idea what a “SSN Ontology” or “SOSA vocabulary” vocabulary are when they’re described in the Scope section. Just some links and things would help!

**Comment by simoneonofri** [comment]
No comments from Security

**Comment by ruoxiran** [comment]
no comments from APA.

**Comment by svgeesus** [comment]
You could delete

> Note:Display this table and update it when appropriate. Requirements for charter extension history are documented in the [Charter Guidebook (section 4)](https://www.w3.org/Guide/process/charter.html#extension).

**Comment by bert-github** [comment]
Deleted.

**Comment by siusin** [comment]
The spec name in the 4th bullet of [the Scope Sec.](https://w3c.github.io/charter-drafts/2024/sdw-wg.html#scope) is missing.

**Comment by bert-github** [comment]
Fixed.

**Comment by plehegar** [comment]
[Proposed W3C Charter: Spatial Data on the Web Working Group (until 2024-12-12/13)](https://lists.w3.org/Archives/Public/public-new-work/2024Nov/0004.html)

**Comment by plehegar** [comment]
[Announced](https://lists.w3.org/Archives/Member/w3c-ac-members/2025JanMar/0030.html)

**Comment by plehegar** [comment]
Note: the Group is now named: Spatio-temporal Data on the Web Working Group
